packages/story-editor/src/components/panels/document/taxonomies/FlatTermSelector.js

```diff
@@ -110,6 +128,15 @@ function FlatTermSelector({ taxonomy, canCreateTerms }) {
     [taxonomy, termCache]
   );
 
+  const suggestedTerms = useMemo(
```

Comment:
added this to clarify and simplify what's expected for the list in `Input` - could pass in `flatSearchResults` as is if that's the preference.